webdriver/tests/bidi/script/call_function/csp_tentative.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+import asyncio
+
+from webdriver.bidi.modules.script import ContextTarget
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "function_declaration",
+    [
+        "function() { return 2 + 1; }",
+        "function() { return eval('2 + 1'); }",
+        "new Function('return 2 + 1')",
+        """
+          function () {
+            return new Promise(r => {
+              setTimeout(() => {
+                r(eval('2 + 1'));
+              }, 0);
+            });
+          }
+        """,
+        """
+          async function() {
+            await new Promise(r => setTimeout(r, 0));
+            return eval("2 + 1");
+          }
+        """,
+        "window.inlineScriptEval",
+        "window.preloadScriptEval",
+        "window.document.body.onclick",
+        "function() { return eval(\"eval('2+1')\") }",
+        "new Function(\"return new Function('return 2 + 1')()\")",
+        "function() { return eval(\"new Function('return 2 + 1')()\") }",
+        "new Function(\"return eval('2+1')\")",
+    ],
+    ids=[
+        "default",
+        "eval",
+        "new Function",
+        "promise eval",
+        "async eval",
+        "eval from inline script",
+        "eval from preload script",
+        "eval from event handler",
+        "nested eval",
+        "nested new Function",
+        "new Function nested in eval",
+        "eval nested in new Function",
+    ],
+)
+async def test_default_src_unsafe_inline(
+    bidi_session, inline, top_context, add_preload_script, function_declaration
+):
+    url = inline(
+        """<!DOCTYPE html>
+          <html lang="en">
+            <head>
+              <meta http-equiv="Content-Security-Policy" content="default-src 'unsafe-inline'">
+            </head>
+            <body onclick="return eval('2 + 1')">
+              <script type="text/javascript">
+                window.inlineScriptEval = function () {
+                  return eval("2 + 1");
+                }
+              </script>
+            </body>
+          </html>""",
+        doctype="html_quirks",
```

Comment:
Isn't this doctype argument oppositional to the DOCTYPE definition? We should remove it, or?

Reply:
`html_quirks` happens to be a blank template with the text/html mime type, which matches what I need here, so that I can have a clean meta tag and a body tag.

Reply:
Sorry my reply actually started a new thread. :( Lets follow-up on that other one.